### ISSUE TITLE
perinnölliset taudit

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#11660]
**Käsitteen tyyppi**

Muutos olemassa olevaan käsitteeseen

**prefabel**

[perinnölliset taudit](http://www.yso.fi/onto/yso/p19997)

**Tila**

Käsittelyssä

**Ehdotettu muutos**

Ohjaustermiksi geneettiset sairaudet

**Perustelut ehdotukselle**

https://termipankki.fi/tepa/fi/haku/geneettinen%20sairaus

**Ehdottajan organisaatio**

**Comment by fantsby-m** [comment]
_Synnynnäiset geneettiset sairaudet_, kuten MeSH?

**Comment by mirmaid** [comment]
Olisi kiinnostavaa tietää, mitä MeSHissä tarkoittaa tuo "inborn" - englanniksi osana päätermiä, suomeksi päätermi vain "perinnölliset". Ehkä MeSHin luokittelussa tämän ajatellaan kattavan ne perinnölliset taudit, joissa sairaus ilmenee jo varhain. Perinnölliset ei kuitenkaan rajaudu (ainakaan YSOssa) näin tiukasti.

**Comment by mariaforsen** [comment]
Kannatan nykyistä muotoa pääterminä (yleiskieltä kun on). Meshin sv-muoto voidaan yleissanaston kontekstissa helposti tulkita merkitykseltään suppeammaksi.

**Comment by FinMeSHienPaimentaja** [comment]
MeSH-määritelmä:

Genetic Diseases, Inborn

https://meshb.nlm.nih.gov/record/ui?ui=D030342

Scope Note:

Diseases that are caused by genetic mutations present during embryo or fetal development, although they may be observed later in life. The mutations may be inherited from a parent's genome or they may be acquired in utero.

**Comment by FinMeSHienPaimentaja** [comment]
Mielestäni ei voi laittaa yhtäläisyysmerkkejä termien "perinnölliset / periytyvät sairaudet"  ja "geneettiset sairaudet" välille, vaikka voivat ollakin limittäin. 

Kaikki geneettiset sairaudet eivät ole perinnöllisiä, mutta intuitiivisestikin kaikki perinnölliset sairaudet ovat geneettisiä. 

Esimerkiksi luomi, joka vanhuudessa muuttuu elämänaikaisen uv-säteilyn aiheuttamien dna-hämminkien takia pahanlaatuiseksi ja syöväksi, on kylläkin pohjimmiltaan geneettinen sairaus, mutta ei perinnöllinen sairaus. 

Mutta tuohon ohjaustermiasiaan minulla ei ole vahvaa ja/tai lukkoonlyötyä kantaa. 

(Juu, ja FinMeSH:it kaipaavat tarkistusta ja/tai päivitystä).

**Comment by YSO-kokous** [comment]
**Päätös**

Tutkitaan vielä muutetaanko päätermiksi _geneettiset sairaudet_ tai _geneettiset taudit_.

**Comment by tpalonen** [comment]
Ks. https://lpamrs.memberclicks.net/assets/documents/Sporadic%20Genetic%20mutations.pdf
"By definition, a sporadic genetic disease is not inherited from parents, but arises via a mutation."

**Comment by AuHo22** [comment]
Tarvitaanko LT geneettiset sairaudet ST perinnölliset taudit? MeSH myös: geneettinen ilmiö KÄ perinnölliset ilmiöt -> näin ei liene myöskään aina ole tai onko käännös asianmukainen?

**Comment by ilona-x** [comment]
TERO:
**perinnölliset taudit** http://www.yso.fi/onto/tero/p19997
Ohjaustermit
    hereditaariset sairaudet
    hereditaariset taudit
    perinnölliset sairaudet
    periytyvät sairaudet
    periytyvät taudit
    synnynnäinen geneettinen sairaus
    synnynnäiset geneettiset sairaudet

**Comment by ilona-x** [comment]
Huom. YSOssa [perinnölliset taudit ](http://www.yso.fi/onto/yso/p19997)-termillä on nyt englanniksi ja ruotsiksi ohjaustermit genetic diseases & genetiska sjukdomar
ja LCSH-linkki genetic disorders

Ja Wikidatassa suomeksi perinnöllinen sairaus ja englanniksi genetic disorder ja ruotsiksi genetisk sjukdom
https://www.wikidata.org/wiki/Q200779

**Comment by AuHo22** [comment]
https://www.veritasint.com/blog/difference-between-genetic-and-hereditary-diseases/

Myös englanninkielistä termiä familial diseases on käytössä, mutta lienee hivenen eri asia sekin:
https://www.obgproject.com/2021/08/30/familial-vs-hereditary-disease-definitions-and-key-counseling-points/

**Comment by tpalonen** [comment]
Perinnöllinen sairaus on geneettistä sairausta selvästi käytetympi muoto Google, yle.fi ja hs.fi -hakujen perusteella.

**Comment by YSO-kokous** [comment]
**Päätös**

Siirretään ensi kokoukseen.

**Comment by mariaforsen** [comment]
Kuten aikaisemmin todettu niin **geneettiset sairaudet** on tässä varmaan tärkein termi. Jos ei haluta kahta termiä (laajempana geneettiset ja suppeampana perinnölliset) niin varmaan perinnölliset taudit pitää olla ohjausterminä. Näin ainakin ruotsin osalta. 

 "Genetiska sjukdomar orsakas av mutationer, det vill säga genetiska förändringar. En del genetiska sjukdomar är hundra procent ärftliga och beror bara på mutationer. Andra är inte ärftliga och orsakas av mutationer som uppstått under livets gång. Till dessa hör vissa former av cancer."
https://www.genteknik.se/genetik-och-genteknik/genmodifierade-organismer-gmo/manniska-och-medicin/genetiska-sjukdomar/

SAOssa vain yksi termi, mutta pääterminä tietysti genetiska sjukdomar: https://id.kb.se/term/sao/Genetiska%20sjukdomar

Nationalencyklopedin: "ärftliga sjukdomar, en undergrupp av genetiska sjukdomar"

**Comment by mirmaid** [comment]
Lääketieteen termit

**perinnöllinen sairaus**: sairaus jonka perussyy on perimän virheessä; ks. myös heredopatia

**heredopatia**: perinnöllinen sairaus, perinnöllinen tauti | sairaus jonka perussyy on sukupolvesta toiseen siirtyvässä perimän virheessä

**geneettinen**: perinnöllisyyteen liittyvä, perinnöllinen

**Comment by mirmaid** [comment]
Ks. myös https://www.terveyskirjasto.fi/dlk00985 (samaa jo Rosa edellä kommentoikin). Suomeksi geneettinen varmaan yleensä tarkoittaa perinnöllistä, en oikein usko että tautiin joka syntyy somaattisen mutaation seurauksena tai geneettisen alttiuden ja ympäristötekijöiden yhteisvaikutuksesta viitattaisiin geneettisenä tautina (?).

**Comment by FinMeSHienPaimentaja** [comment]
Joo, monitekijäisissä taudeissa, joissa siis geneettiset syyt ovat vain yksi osatekijä, on kätevä etenkin olemassaoleva yso-termi "geneettiset tekijät". Myös "perinnöllinen alttius" on ysona. 

MeSH-aspektiterminähän on myös *genetics, mikä niin ikään on kätevä, jos ja kun sen voi liittää monen sairauden aspekti/näkökulmatermiksi.

Melkein aina näitä asiasanoitettaessa on kyse jostain tietystä taudista (tai tautiryhmästä) - geneettiset taudit tai sairaudet (tai perinnölliset/periytyvät taudit) yleisesti ottaen tulee vastaan aika harvoin (jos on asiasanoitettu oikein), lähinnä lääketieteellisen genetiikan oppikirjoissa tai vastaavissa yleisteoksissa tullee vastaan.

**Comment by YSO-kokous** [comment]
YSO-kokous päätti säilyttää käsitteen pääterminä perinnölliset taudit.
Lisätään ohjaustermi geneettiset taudit (perinnölliset taudit) KÄYTÄ perinnölliset taudit.

Asiasta äänestettiin YSO-kokouksessa. Enemmistö kannatti päätermiä perinnölliset taudit (vs. geneettiset taudit)

Tarkistetaan muunkieliset termit. YSO-ylläpito selvittää, onko ohjaustermi geneettiset taudit KÄYTÄ geneettiset tekijät tarpeellinen. Tai käyttöhuomautus perinnöllisiin tauteihin, jossa ohjaus geneettisiin tauteihin.

**Comment by mariaforsen** [comment]
Säilytetäänkö linkki LCSH:iin vaikkei merkityskenttä olekaan aivan sama? On ehkä syytä pitää sama merkityskenttä (perinnöllinen, ei geneettinen) myös englanniksi ja ruotsiksi, eli ei muutoksia näihin päätermeihin?
